### PR TITLE
Fix nextflow pipelines bug with cluster options and prio

### PR DESCRIPTION
**Pull request by rannick**
## Description

Priorities and cluster options were not written in params_file of nextflow pipelines rnafusion and taxprofiler, causing them to fail with command run

### Added

-

### Changed

-

### Fixed

- nextflow pipelines bug with cluster options and prio introduced in https://github.com/Clinical-Genomics/cg/pull/2725/files#

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/models/rnafusion/rnafusion.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ class RnafusionParameters(WorkflowParameters):
     fusioncatcher: bool = True
     starfusion: bool = True
     trim_tail: int = 50
+    clusterOptions: str = Field(..., alias="cluster_options")
```
Comment (henrikstranneheim) [inline cg/models/rnafusion/rnafusion.py:41]: Can't we use snake case here?

File: cg/models/taxprofiler/taxprofiler.py [hunk 2]
```diff
@@ -39,6 +39,8 @@ class TaxprofilerParameters(WorkflowParameters):
     centrifuge_save_reads: bool = True
     run_krona: bool = True
     run_profile_standardisation: bool = True
+    clusterOptions: str = Field(..., alias="cluster_options")
```
Comment (henrikstranneheim) [inline cg/models/taxprofiler/taxprofiler.py:42]: And here

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by rannick** [comment]
I agree, but I would like to test any change extensively so I will address that in another PR next week and merge this already

**Comment by rannick** [comment]
Screenshot 2024-03-11 at 08 52 26

**Pull request merged**